website/i18n/en.json

```diff
@@ -257,6 +257,9 @@
       "platform-specific-code": {
         "title": "Platform Specific Code"
       },
+      "platformcolor": {
```

Comment:
I think we should be consistent between PlatformColor and DynamicColorIOS. They are both public APIs from React Native, and they can both be used everywhere any other color can. So if one has a standalone page, they should both have a standalone page. If one is in the colors.md file, they both should be.

Reply:
I'm a bit confused. I got the impression Rachel wanted dynamiccolorios in the colors.md and the platformcolor separate.

@rachelnabors - Should I pull out the dynamiccolorios and put it in APIs as a separate section, or combine both into colors.md?

Reply:
I'm curious what Rachel will say here, but I'm not sure why `DynamicColorIOS` would be treated differently than `PlatformColor` since they are the essentially the same.